Program.cs

```diff
@@ -1,16 +1,161 @@
+        List<int> lesser10andnegative = integers2.Where(i => i < 10).ToList();
+        foreach (var v in lesser10andnegative)
+        {
+            Console.WriteLine("Less than 10 and negative: {0}", v);
+        }
+        int[] integers2 = new[] { 15, -8, -7, -5, 6, 14, 13, -9, 11, -10 };
+        List<int> lesser10andnegative = integers2.Where(i => i < 10).ToList();
+        foreach (var v in lesser10andnegative)
+        {
+            Console.WriteLine("Less than 10 and negative: {0}", v);
+        }
+
+        #endregion
+
+        #region Task3
+
+        List<string> brands = new List<string> { "chanel", "dior", "gucci", "stella mccartni", "adidas" };
+        var filteredbrands = from b in brands
+            where Helpers.CheckIfIsDevidedWithoutRamainder(b.Length, 2) ==
+                  true // b.Length== Helpers.CheckIfIsDevidedWithoutRamainder(b.Length, 2)
+            orderby b
+            select b;
+        foreach (var v in filteredbrands)
+        {
+            Console.WriteLine("Brands have even length: {0}", v);
+        }
+
+        #endregion
+
+        #region Task4
+
+        List<string> cities = new List<string>() { "Berlin", "London", "Paris", "Bobruisk", "Tokyo" };
+        var citiesfiltered = cities.Select(i => i).Where(i => i.StartsWith("B") && i.EndsWith("k")).ToList();
+
+        foreach (var g in citiesfiltered)
+        {
+            Console.WriteLine("City of dream is {0}", g);
+        }
+
+        #endregion
+
+        #region Task5
+        //couldn't solve, here my drafts
+
+        //Console.WriteLine("input string with words in different case");
+        //string allstring = Console.ReadLine();
+        //allstring.Substring(0,Indexof(Indexof()))
+        //   IsLower(allstring,)
+
+        //       if()
+        // List<string> allcases = new List<string>() { "BERLIN", "london", "paris", "BOBRUISK" ,"TOKYO" };
+
+        #endregion
+
+        #region Task6 
+        //couldn't solve yet, here drafts
+        int[] integers3 = new[] { 15, 4, -7, 15, -7, 14, 13, 4, 11, 15 };
+        /*var onlydistinct = integers3.Where(i => i.Distinct().ToList();
+        int[] reversedlist = onlydistinct.Reverse();
+        integers2.Distinct();*/
+
+        #endregion
+
+
+        #region Task7
+
+        bool checkIfTwo = IsTheNumberInMassive(2, integers);
+        Console.WriteLine($"Is  in massive?: {checkIfTwo}");
+
+        #endregion
+
+        #region Task8
+
+        /*List<string> twocolor = new List<string>();
+
+        List<string> tricolor = new List<string>() { "#FFFFFF", "FF0000", "#FFFFFF" };*/
+        //var selectedcolors = (tricolor.Count > 5).select(tricolor.First, tricolor.Last(a=> i)).ToList();
+
+        /*var selectedcolors = from t in tricolor 
+        where GetListAllElementsHeight(tricolor) > 5
+        select (tricolor.SelectMany(tricolor.First(), tricolor.Last()));*/// not sure how to select 1st and 3rd elements
```

Comment:
tricolor.First(), tricolor.Last() -  yes, this is correct

Reply:
fixed

Reply:
fixed